compile.js

```diff
@@ -95,7 +97,8 @@
     }
 
     delete flickr.options;
-    buffer.write("(function() {");
+    // buffer.write("(function() {");
+    buffer.write(wrapper[0]);
```

Comment:
for legibility, can we capture `wrapper[0]` and `wrapper[1]` with named variables to make it obvious what they represent?